pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -50,7 +50,7 @@ var (
 	securityGroup     = securityapi.GroupName
 	storageGroup      = storage.GroupName
 	authzGroup        = authorizationapi.GroupName
-	buildGroup        = buildapi.GroupName
+	buildGroup        = buildapi.LegacyGroupName
```

Comment:
ah, you want both.  Might make `buildGroups` and use it below. `Groups(buildGroups...)`